source/docs/software/telemetry/persistent-alerts.rst

```diff
@@ -0,0 +1,55 @@
+# Recording Faults with Persistent Alerts
+
+Robots encounter a variety of fault conditions: disconnected sensors/motors, invalid mechanism states, initialization failures, etc. While the FRC Driver Station provides a :ref:`console interface <docs/software/driverstation/driver-station:Messages Tab>` for instantaneous alerts, the risk of missing important messages makes it poorly suited to communicate faults that persistent over time.
+
+Instead, WPILib provides an API for managing persistent alerts published via Network Tables. Alerts are assigned a priority (*error*, *warning*, or *info*) and can be *activated* or *deactivated* in robot code. The set of active alerts can be displayed on a dashboard, as shown below in Shuffleboard.
```

Comment:
I think this would be a good place to link to the API docs for this.  Especially as we later reference them as the ideal way to learn about the alert levels

Reply:
See [here](https://docs.wpilib.org/en/latest/docs/software/advanced-controls/controllers/pidcontroller.html) for an example of linking to the API.